WordPress/Classes/ViewRelated/Media/External/ExternalMediaPickerViewController.swift

```diff
@@ -86,7 +88,7 @@ final class ExternalMediaPickerViewController: UIViewController, UICollectionVie
     }
 
     private func configureCollectionView() {
-        collectionView.register(cell: ExternalMediaPickerCollectionCell.self)
+        collectionView.register(ExternalMediaPickerCollectionCell.self, forCellWithReuseIdentifier: Self.cellReuseID)
```

Comment:
This method was coming from `Kanvas`. I'm not sure why it was working without the `import` – pch, ObjC briding header?